articles/logs/streams/azure-event-grid.md

```diff
@@ -0,0 +1,98 @@
+---
+title: Azure Event Grid Log Streams
+description: Learn how to create an event-driven workflow using Azure Event Grid and send your tenant logs anywhere within the Azure ecosystem.
+toc: false
+topics:
+ - logs
+ - streams
+ - event-streams
+contentType: how-to
+---
+
+# Integrate Azure Event Grid with Auth0
+
+Azure Event Grid is a serverless event bus that lets you send event data from any source to any destination.
+
+You can create event-driven workflows using to Event Grid send your Auth0 tenant logs to targets such as Azure Functions, Event Hubs, Sentinel, and Logic Apps.
+
+See the Auth0 event type codes for a [full list](https://auth0.com/docs/logs/references/log-event-type-codes) of the events that Auth0 supports.
+
+## Send events from Auth0 to Azure Event Grid
+
+To send Auth0 events to Azure, you'll need to:
+
+1. Enable Event Grid resource provider.
+2. Set up an event source (in this case, this is Auth0).
+3. Set up an event handler, the app or service where the event will be sent.
+
+For more information about these concepts, [see Concepts in Azure Event Grid](https://docs.microsoft.com/en-us/azure/event-grid/concepts)
+
+### Enable Event Grid resource provider
+
+If you haven’t previously used Event Grid, you will need to register the Event Grid resource provider. If you've used Event Grid before, skip to the next section.
+
+In your Azure portal:
+
+1. Select Subscriptions on the left menu
+2. Select the subscription you’re using for Event Grid
+3. On the left menu, under Settings, select Resource providers
+4. Find Microsoft.EventGrid
+5. Select Register
+6. Refresh to make sure the status changes to Registered
+
+### Set up an Auth0 event source
+
+Part of the integration process is to set Auth0 up for use as an event source (this step happens on your Dashboard).
+Log in to the Auth0 Dashboard.
+
+1. Navigate to Logs > Streams.
+2. Click + Create Stream.
+3. Select Azure Event Grid and enter a unique name for your new stream.
+4. Create the event source by providing your Azure Subscription ID, Azure Region and a Resource Group name. 
```

Comment:
On the Datadog and HTTP pages, we use a table to describe each of these fields. Maybe we could do that here as well?